manifests/init.pp

```diff
@@ -666,6 +664,10 @@
     $_purge_mutators   = $full_purge_hash['mutators']
   }
 
+  # (#463) This well-known anchor serves as a reference point so all checks are
+  # managed after all plugins.  It must always exist in the catalog.
+  anchor { 'plugins_before_checks': }
```

Comment:
We want to kill off the anchor pattern in the next big refactor after the data types. Is there another way we can do this, such as using the spaceship operator?